Geometry/HcalCommonData/python/hcalDDDSimConstants_cff.py

```diff
@@ -2,6 +2,6 @@
 
 from Geometry.HcalCommonData.hcalParameters_cff             import *
 from Geometry.HcalCommonData.hcalSimulationParameters_cff   import *
-from Geometry.HcalCommonData.hcalDDDSimConstants_cfi        import *
 from Geometry.HcalCommonData.hcalSimulationConstants_cfi    import *
 from Geometry.HcalCommonData.caloSimulationParameters_cff   import *
+from Geometry.HcalCommonData.hcalDDDSimConstants_cfi        import *
```

Comment:
Is the order of the cfi's important?

Reply:
No it is not.

Reply:
Then, unless my eyes are failing me, there is no actual change to this file, and it should be removed from the PR.

Reply:
What is the conclusion on this question? Thanks.

Reply:
I have the ordering to be similar in all cff files. That is why it is changed here.

Reply:
OK from my side. This is up to @cvuosalo.

Thanks.